src/Theme.js

```diff
@@ -20,6 +20,13 @@ const darkTheme = createTheme({
         },
       },
     },
+    Box: {
+      styleOverrides: {
+        root: {
+          // background: "transparent",
```

Comment:
There was a rendering issue with the search bar not responding, thus went down the rabbit hole to find which would control it. I believe that was an outdated solution so it could be removed.